tests/integration/test_runtime.py

```diff
@@ -15,7 +15,7 @@
 logger = logging.getLogger(__name__)
 
 
-@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=5))
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=20))
```

Comment:
This is getting long. I understand it is a "do not blame the messenger", the changes in this PR do not make the test run longer, but make it visible that it runs longer. Still something to keep an eye on

Reply:
Agreed.

And often this is an indication of a problem with something under test rather than the  test itself. In particular there's normally something within the end-to-end flow that needs to be configured for retry with a (much smaller!) timeout. However because that's missing the whole test fails and therefore the whole thing needs to be restarted from scratch.